Cargo.toml

```diff
@@ -95,8 +96,9 @@ fontconfig-dlopen = ["font-kit/source-fontconfig-dlopen"]
 
 # Misc
 datetime = ["chrono"]
-evcxr = ["svg_backend"]
+evcxr = ["svg_backend", "bitmap_backend", "image"]
```

Comment:
And I think this means once evcxr feature is on then bitmap_backend will be compiled. And I feel this is an additional burden